blockly-core/core/ui/block.js

```diff
@@ -493,11 +493,24 @@ Blockly.Block.prototype.moveBy = function(dx, dy) {
  * @return {!Object} Object with height and width properties.
  */
 Blockly.Block.prototype.getHeightWidth = function() {
+  var bBox;
+
   try {
-    if (Blockly.ieVersion() && Blockly.ieVersion() <= 10) {
-      this.getSvgRoot().style.display = "inline";   /* reqd for IE */
+    var ieLessThan10 = Blockly.ieVersion() && Blockly.ieVersion() <= 10;
+    var initialStyle;
+
+    if (ieLessThan10) {
+      // Required to set display to inline during calculation in IE <= 10
+      initialStyle = this.getSvgRoot().style.display;
+      this.getSvgRoot().style.display = "inline";
+    }
+
+    bBox = goog.object.clone(this.getSvgRoot().getBBox());
+
+    if (ieLessThan10) {
+      // Reset to original display value
+      this.getSvgRoot().style.display = initialStyle;
```

Comment:
Hm, I'm a little wary of making a change that would tweak the existing code path for non-ie9/10 browsers and potentially be dependent on each browser's `style.display`'s set and get behavior (though it might be the case that it would always be a noop in all browsers, I'm just not 100% confident)